routes/set_password.py

```diff
@@ -0,0 +1,59 @@
+from flask import Blueprint, request
+from flask_api import status
+
+from models.user import User, ActiveType
+from util.error import Error
+from util.database import db
+
+SetPassword = Blueprint('SetPassword', __name__)
+
+
+class Errors:
+    UNKNOWN_USER = Error('Unknown user', 'No user found')
+    WRONG_TOKEN = Error('Wrong token', 'Check token and try again.')
+    NO_ACTIVATION_PENDING = Error('No Activation pending',
+                                  'There is no activation pending for this user.')
+    PASSWORD_TOO_SHOORT = Error('Password too short',
+                                'Provide a longer password and try again')
+
+
+@SetPassword.route('/set-password', methods=['POST'])
+def setPassword():
+    """ Set the password for an user who's activation is still pending.
+
+    Parameters:
+      String -- user: The target user's name.
+      String -- token: The target user's token.
+      String -- password: The target user's password in plaintext to set.
```

Comment:
Do we really want to transfer the password in plain text? If it is encrypted at a different place, where?

Reply:
Plaintext just refers to the parameter.